kpatch-build/kpatch-build

```diff
@@ -462,6 +463,12 @@ do
 	cp -f "$OBJDIR/$i" "$TEMPDIR/patched/$i" || die
 done
 
+# Detect patching type
+if grep -q "CONFIG_LIVEPATCH=y" "$CONFIGFILE"; then
+  PATCHTYPE="klp"
```

Comment:
I'd prefer to s/klp/livepatch/ and s/KLP/LIVEPATCH/, since it's not obvious to all what klp stands for.

Also, use tabs for indentation to be consistent with the rest of the file.